ports/sonictt16b/port.json

```
@@ -0,0 +1,29 @@
+{
+  "version": 2,
+  "name": "sonictt16b.zip",
+  "items": [
+    "Sonic Triple Trouble - 16-Bit.sh",
+    "sonictt16b/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Sonic Triple Trouble - 16-Bit",
+    "porter": [
+      "Jeod",
+	  "Jantrueno"
+    ],
+    "desc": "What if Sonic Triple Trouble had been originally released on the Sega Genesis / MegaDrive, after Sonic 3 &amp; Knuckles?",
+    "inst": "Download the 64-bit android apk from Gamejolt and put it in `ports/sonictt16b` as `game.apk`.",
+    "genres": [
+      "action",
+      "adventure",
+      "arcade",
+      "platformer"
+    ],
+    "image": "screenshot.png",
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": ["aarch64"]
```

This should be in the `"arch"` tag.
```suggestion
    "reqs": [],
    "arch": ["aarch64"]
```

---

The requirements attribute isn't for filtering architecture.